pkg/status/README.md

```diff
@@ -0,0 +1,12 @@
+# pkg/status
+
+## Overview
+
+This module contains functionality for aggregating status information about components. Used by extensions that
+implement the `componentstatus.Watcher` interface for collecting status information about the internal status of all
+components inside the collector.
+
+### History
+
+`status` was originally developed for the `healthcheckv2extension`. Other extensions that need to collector status
```

Comment:
Can you double check the wording on this second sentence?

Reply:
Not a blocker, but could you take a look at this @blakerouse?